docs/src/plasma/qtplasmac.adoc

```diff
@@ -968,15 +956,15 @@ The torch (T0) will be selected if it was not the active tool.
 |RELOAD |This button will reload the currently loaded G-code File.
 |===
 
-[underline]*DRO*
-
-[width="100%",cols="4,16",options="header"]
+.*THC*
```

Comment:
Why did that change?

Reply:
Some merge gone wild, I presume. Thank you for spotting that. .THC was above.